app/helpers/support_interface_helper.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module SupportInterfaceHelper
+  def user_info_link(user)
+    govuk_link_to(
+      [user["firstName"], user["lastName"]].join(" "),
```

Comment:
Should we create a wrapper class for the user object? That would allow us to only reference "firstName" in only one place, making it easier to deal with future API changes, and also would be a good place to place `def full_name` helpers and such.

Reply:
Agree with this — alluded to it briefly in my [show page PR](https://github.com/DFE-Digital/find-a-lost-trn/pull/483), but we could have something like an `app/models/user.rb` that accepts the JSON returned by the API in its constructor.

(Could live somewhere other than "model" if that's likely to be confusing at all).